appengine/standard_python37/pubsub/data/privatekey.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Are you sure you want to check these in, rather than explain how to generate them in the README?

Reply:
Yes, I do want to check them in for testing purposes. :) Users shouldn't need to generate these when using authenticated push.